inventory-countdown/template/src/site/plugins/custom-elements/stock-counter/plugin.tsx

```diff
@@ -31,20 +31,24 @@ function getInventoryStock(productId: string): Promise<InventoryStock> {
         },
       })
       .then((result) => {
-        const productInventory = result.inventoryItems[0];
+        const productInventory = result?.inventoryItems && result.inventoryItems[0];
```

Comment:
The moment I tighten up types - these types now become aligned with what we actually get from the server.
Basically all properties of "result" are undefined.